### PR TITLE
Update pgrx, add postgres 16

### PR DESCRIPTION
**Pull request by maparent**

### CODE REVIEW

**Comment by maparent** [comment]
Note: Solves #1088

**Review by kczimm — CHANGES_REQUESTED** [review]

**Comment by levkk** [comment]
Would love to have this since Postgres just removed 16.rc1 from its FTP repo and we are breaking on `pgrx init` using 0.10.0.

File: .github/workflows/ubuntu-packages-and-docker-image.yml [hunk 1]
```diff
@@ -72,12 +72,13 @@ jobs:
           libpq-dev \
           libclang-dev \
           wget \
+          postgresql-16 \
           postgresql-15 \
           postgresql-14 \
           postgresql-13 \
           postgresql-12 \
           postgresql-11 \
-          postgresql-server-dev-15 \
```
[the remark this answers is not in the document]
Comment (maparent) [reply .github/workflows/ubuntu-packages-and-docker-image.yml:80]: apologies, near-typo.

**Comment by levkk** [comment]
@maparent I'll take us all the way to pgrx 0.11.0. This will require some handholding with our CI, since you can't really our Github actions locally (requires access to our APT repo and such).

https://github.com/postgresml/postgresml/pull/1099

Thank you for the PR, mine is largely inspired by yours.

**Comment by maparent** [comment]
Superseded by https://github.com/postgresml/postgresml/pull/1102

**Pull request closed without merging**